app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt

```diff
@@ -721,6 +726,13 @@ class ReaderActivity : BaseActivity() {
         startActivity(Intent.createChooser(intent, stringResource(MR.strings.action_share)))
     }
 
+    private fun onCopyImageResult(uri: Uri) {
+        // Copy the URI to the clipboard
+        val clipboardManager = applicationContext.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
```

Comment:
will require an import
```suggestion
        val clipboardManager = applicationContext.getSystemService<ClipboardManager>()
```